company/okrs/2020_q1.md

```diff
@@ -0,0 +1,69 @@
+
+## Sales and marketing
+
+1. **Sales (Closing Deals): Achieve [$N][N] net new ARR.**
+   1. Sales KR: Have each AE close 3 Tier 1 deals (>=$100K ARR) and 10 non-Tier 1 deals, 2 of which are from their own outbound outreach efforts.
+   1. Sales KR: Get 70% of companies on a free trial to convert to paid customers.
+1. **Sales (Pipeline Optimization): Scale our sales pipeline to 50+ deals per AE, of which 60% of deals are engaged and actively evaluating Sourcegraph.**
+   1. Sales KR: Round robin new instances across team to collectively achieve a 15%+ response rate from Prospects who set up a new Sourcegraph instance.
+
+1. **VP Business: Build a Customer Success organization that supports and helps us exceed our ARR goals.**
+   1. VP Business (Success) KR: Generate at least [$M][M] in net new ARR from existing accounts.
+   1. VP Business (Success) KR: Hire a Head of Customer Success who can own this function.
+   1. VP Business (Success) KR: Reach monthly NPS of 60 at all Tier 1 customers.
+   
+1. **VP Business: Make Sourcegraph a lean organization that uses budgets to guide decision making.**
+
+1. **VP Marketing: Deliver sufficient MQLs to sales to reach [$N][N] net new ARR goal.**
+1. **VP Marketing: Create new category of developer tool.**
+1. **VP Marketing: Become the dominant provider of source code search.**
+
+## Product
+
+1. **Product: WIP Deliver on product roadmap**
+   1. Product KR: Roadmap communicates rough timeline for upcoming work.
+   1. Product KR: Build out per-team strategy RFCs to buffer roadmap.
+   1. Product KR: Dogfooding each feature release to ensure quality UX.
+1. **Product: WIP Reduce product friction**
+   1. Product KR: Reduce time to "aha" (0 to search).
+   1. Product KR: Reduce time 0 to deploy.
+   1. Product KR: Improve out of the box experience.
+1. **Product: WIP Grow product team**
+   1. Product KR: Hire to fill position gaps (1 UX Designer, 1 PM).
+   1. Product KR: Create handbook section outlining product strategy and operation guide.
+   1. Product KR: Create scalable customer feature prioritization.
+
+## Engineering
+
+1. **VP Eng: Build a world class engineering team.**
+   1. VP Eng KR: Grow team to plan (+1 code intel, +2 distribution, +2 web, +1 core services).
+   1. VP Eng KR: Transition an existing teammate to be (or hire) an Engineering Manager of the distribution team.
+   1. VP Eng KR: Document how we do performance reviews and determine compensation for teammates in engineering roles.
+1. **VP Eng: Create a data driven planning process.**
+   1. VP Eng KR: Each item in a tracking issue has an estimate of time or size.
+   1. VP Eng KR: Each team commits to a written work plan for each release cycle that accounts for the work estimates and the team's capacity.
+   1. VP Eng KR: Each team completes >80% of the work that they committed to at the beginning of the release cycle each iteration.
+
+1. **Distribution: Resolve issues for critical customer deals and expansions**
```

Comment:
@beyang What is at least 1 KR for this? How will we know if we succeeded?

Reply:
How about "Resolve all deal-blocking support requests for tier 1 and tier 2 customers"?

@dadlerj is there a way to tag support conversations as "deal-blocking" in Hubspot?
@juliasourceress are we continuing with our bucketing of customers into tiers 1, 2, and 3 this quarter?